core-services/src/main/java/org/zalando/nakadi/service/timeline/MultiTimelineEventConsumer.java

```diff
@@ -252,8 +252,10 @@ private void electTopicRepositories() throws InvalidCursorException {
                 final TopicRepository repo = entry.getKey();
                 LOG.info("Creating underlying consumer for client id {} and cursors {}",
                         clientId, Arrays.deepToString(entry.getValue().toArray()));
+
+                final String eventTypeName = entry.getValue().get(0).getEventType(); // FIXME
```

Comment:
it looks like there is a better place to get the event type name or Kafka record deserialiser